tests/support/foo-resolve/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+	"version": "1.6.2",
```

Comment:
I know the other tests also say version 1.6.2. I'm wondering if we want to update all of them to 2.1.x?